last_5_so_po/models/product_template.py

```diff
@@ -0,0 +1,46 @@
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    # Added fields #T00447
+    last_5_po_ids = fields.One2many(
+        comodel_name="purchase.order",
+        inverse_name="product_id",
+        compute="_compute_last_5_po",
+        string="Purchase orders",
+    )
+
+    last_5_so_ids = fields.One2many(
+        comodel_name="sale.order",
+        inverse_name="product_id",
+        compute="_compute_last_5_so",
+        string="Sale orders",
+    )
+
+    def _compute_last_5_po(self):
```

Comment:
Method name should be _compute_last_five_purchase_order, do not use numeric in method name